server.js

```diff
@@ -13,13 +14,19 @@ app.use(function(req, res, next) {
   res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
   next();
 });
+app.use(express.static(path.join(__dirname, "build")))
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.set('port', process.env.PORT || 3000);
 app.listen(app.get('port'), () => {
   console.log(`We running on ${app.get('port')}.`)
 })
 
+//display something at the root
```

Comment:
May want to add webpack hot loading here, so we don't have to rebuild manually and restart the server!